### ISSUE TITLE
Add documentation about pages, navigation and transitions

### ISSUE THREAD

**Issue by jaspermdegroot** [jquery/api.jquerymobile.com#146]
Most of this section in our old demos/docs didn't land in the API documentation yet: http://view.jquerymobile.com/old-docs/docs/pages/index.html

**Comment by agcolom** [comment]
@jaspermdegroot the original old docs link goes to 404... I tried to look around... is it the content here that need to be brought to the api docs?
http://view.jquerymobile.com/1.2.1/docs/pages/index.html
Thanks

**Comment by jaspermdegroot** [comment]
@agcolom 

Yeah, there is a lot of info in that section of the 1.2 docs that never made it to the API docs. I think some examples are single vs. multi-page, prefetching, ```data-title```.
I think we should go over all subsections and see if the info is already documented somewhere in the API docs and if not see where we should at in the API docs. Not everything might be applicable anymore or the info needs to be updated first.
We could create separate tickets for each subsection or add a checklist for all of them in this ticket.
